runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -36,11 +36,40 @@
 {
 	String GROUP_NAME = "runelite";
 
+	@ConfigSection(
+		name = "Window Settings",
```

Comment:
The word "Settings" is used here, but "Configuration" and "Options" are used in other configs. Of these choices I prefer "Settings" personally, but would like for one word or the other to be used in all cases rather than have a mix of them.